tasks/src/text-to-video/about.md

```diff
@@ -1,72 +1,38 @@
-Text-to-video models can generate videos from a given text prompt. They can be used in any application that requires generating temporally and spatially consistent sequence of images from text.  
-
-
 ## Use Cases
 
 ### Script-based Video Generation
-
 Text-to-video models can be used to create short-form video content from a provided text script. These models can be used to create engaging and informative marketing videos. For example, a company could use a text-to-video model to create a video that explains how their product works or that shows how their product can solve a customer's problem.
 
-
 ### Content format conversion
-
 Text-to-video models can be used to generate videos from long-form text, including blog posts, articles, and text files. Text-to-video models can be used to create educational videos that are more engaging and interactive. An example of this is creating a video that explains a complex concept from an article.
 
-
 ### Disaster Relief and Safety Protocols
-
 Text-to-video models can be used to generate videos that show people how to stay safe during a hurricane or how to find food and water after a natural disaster.
 
-
 ### Voice-overs and Speech
-
 Text-to-video models can be used to create an AI newscaster to deliver daily news, or for a film-maker to create a short film or a music video.
 
 
-
-
 ## Task Variants
-
 Text-to-video models have different variants based on inputs and outputs.
 
-
 ### Text-to-video Retrieval

```

Comment:
Is this really using `text-to-video` models though? This is closer to semantic search/sentence similarity.

Reply:
I think you can use a text-to-video model in sort of feature extraction-y fashion and that's what they mention (sort of like semantic similarity)

Reply:
But this sounds more like a different task then (kinda like CLIP)